### ISSUE TITLE
Can a macro load a bunch of files into NPP?

### ISSUE THREAD

**Issue by decrepit-old-geekoid** [notepad-plus-plus/notepad-plus-plus#2116]
I _often_ need to load a good number of files into NPP . These are the same files in the same order every time. To complicate matters I need to load a number of files and then start  a new document, and then load in another file after that.

I have been using a list of these files in a plain text document to allow me to manually load them all, the order in which they are loaded is important. I copy the file and path string and paste it into the "open" dialogue in NPP.

It's kind of tedious to hand load all of these files every time. If I could find a way to invoke a macro to get this chore done it would be good.

I tried just going through the process of loading them all by hand while recording what I was doing as a macro. That didn't work.

I saw elsewhere up here about the shortcuts.xml file and I do not see that macro name in the shortcuts.xml file. I do see the macro name I'd given my recent effort in the list of macros in NPP, but that listed macro doesn't do anything at all when I try to play it back.

Can a macro be made to do this? If not is there any other way to do this?

I'm in Windows Vista at this time, so any alternate means of doing what I want to do would have to take that into consideration.

Thanks for any help.

PS - I think this must be the third time I've had to sign up anew for the NPP forums. I've been here before, from years ago, but now it's all switched over to Github, I see. Hard to follow these NPP forums at times.

**Comment by dail8859** [comment]
> is there any other way to do this?

1. Open Files
1. `File > Save Session...`
1. Close N++
1. Reopen N++
1. `File > Load Session...`

**Comment by decrepit-old-geekoid** [comment]
Thanks for the reply. This almost does what I need, but not quite. Is there any way for it to leave a blank file in between some of the other files I open this way?

I suppose that I could name some new documents something relatively random and just be careful to leave them empty, and then save it all as a session. I'll try that.

**Comment by decrepit-old-geekoid** [comment]
Okay, that _did_ work! Thanks again for your help.
